src/Azure.Functions.Cli/Common/Utilities.cs

```diff
@@ -51,6 +52,12 @@ internal static void PrintVersion()
                 .WriteLine($"Function Runtime Version: {ScriptHost.Version}\n".DarkGray());
         }
 
+        internal static void PrintUpgradeWarning()
+        {
+            ColoredConsole
+                .WriteLine(ErrorColor("You are using an outdated version of the Azure Functions Core Tools. For more information, please see: https://aka.ms/func-v2-upgrade\n"));
```

Comment:
`WriteLine` should handle new lines appropriately for all OSes. If you need an additional line here, please use `Environment.NewLine` instead.